plugin/cmd/kube-scheduler/app/configurator.go

```diff
@@ -186,3 +191,113 @@ func (sc schedulerConfigurator) Create() (*scheduler.Config, error) {
 
 	return sc.CreateFromConfig(*policy)
 }
+
+func (sc *schedulerConfigurator) SetupPolicyConfigMapEventHandlers(client clientset.Interface, informerFactory informers.SharedInformerFactory) {
+	// selector targets only the scheduler's policy ConfigMap.
+	selector := cache.NewListWatchFromClient(client.CoreV1().RESTClient(), "configmaps", sc.policyConfigMapNamespace, fields.OneTermEqualSelector(api.ObjectNameField, string(sc.policyConfigMap)))
+
+	sharedIndexInformer := informerFactory.InformerFor(&v1.ConfigMap{}, func(client clientset.Interface, resyncPeriod time.Duration) cache.SharedIndexInformer {
+		sharedIndexInformer := cache.NewSharedIndexInformer(
+			selector,
+			&v1.ConfigMap{},
+			resyncPeriod,
+			cache.Indexers{cache.NamespaceIndex: cache.MetaNamespaceIndexFunc},
+		)
+		return sharedIndexInformer
+	})
+
+	sharedIndexInformer.AddEventHandlerWithResyncPeriod(
+		cache.ResourceEventHandlerFuncs {
+			AddFunc:    nil, // scheduler aborts if its policy ConfigMap does not exist. So, we do not need and "add" handler.
+			UpdateFunc: sc.updatePolicyConfigMap,
+			DeleteFunc: sc.deletePolicyConfigMap,
+		},
+		0,
+	)
+}
+
+// verifyNewPolicyConfig verifies that the new object received by the ConfigMap watch is a valid policy config.
```

Comment:
We had verified the ConfigMap on startup; if failed, scheduler exit. One purpose of this check is to avoid keep restarting scheduler when ConfigMap invalid after updating.

Anyway, I think we should add an event (and log) to show whether new config was accepted.